app/tests/integration/conftest.py

```diff
@@ -103,89 +105,92 @@ def fill_timeline_page(selenium, base_url, request, variables, fill_overview):
 
 @pytest.fixture
 def fill_design_page(selenium, base_url, request, variables, fill_overview):
```

Comment:
Is this for Generic?   Can you specify in the name which type this is for?

Reply:
Yeah Generic, I'll add that to it